test/passing/source.ml.ref

```diff
@@ -9223,3 +9223,6 @@ type t = |
 M.(Some x) [@foo]
 
 [@@@foo:]
+
+;;
+(x :> t) [@a]
```

Comment:
why is this passing? the source and source.ref are differents!

Reply:
when there is a ref file, it checks that formatting of x.ml is equal to x.ml.ref.

Reply:
I moved this to attributes.ml which as no ref, seems clearer.